src/modules/PopUp.js

```diff
@@ -24,7 +24,6 @@ const clickHandler = (coin) => {
   fetchCoinData(coin)
     .then((coinData) => {
       popupInner.innerHTML = '';
```

Comment:
https://github.com/damdafayton/capstone-javascript/blob/8a6ba0d164e128ab1e88d1d3e47678127e957c9e/src/modules/PopUp.js#L20-L26

- [OPTIONAL] Great job so far 👍 your App works as expected but in the project requirements you are to use `async` and `await` JavaScript features to consume the API but you have implemented your features using `promises`. Kindly rectify this

Reply:
Video representation:
https://drive.google.com/file/d/1-xuiTCdHtBemcroUUKqWTBp7AIWeNW0O/view?usp=sharing